tests/Application/Kernel.php

```diff
@@ -99,6 +100,18 @@ private function registerBundlesFromFile(string $bundlesFile): iterable
     {
         $contents = require $bundlesFile;
 
+        if (SyliusKernel::MINOR_VERSION > 11) {
+            $contents = array_merge(
+                ['League\FlysystemBundle\FlysystemBundle' => ['all' => true]],
+                $contents,
+            );
+        } else {
+            $contents = array_merge(
+                ['Symfony\Bundle\SwiftmailerBundle\SwiftmailerBundle' => ['all' => true]],
+                $contents,
+            );
+        }
+
```

Comment:
I think putting this inside `config/bundles.php` is better as we don't have a feeling of "magically" loaded bundles.

Reply:
I was considering it but we used to do it in the Kernel at least a few times and because of it, I would expect to find such a magic in the Kernel :P

Reply:
I'm ok with that, but I think we should talk about it in our daily meeting. I went `bundles.php` way on PPP and I would like to avoid different approaches across our plugins.